tutorial/4_hextof_workflow.ipynb

```diff
@@ -62,11 +88,199 @@
-    "    \"data_parquet_dir\": \"/home/agustsss/temp/sed_parquet/\"\n",
-    "}}}\n",
-    "sp = SedProcessor(runs=[44797], config=config, user_config=config_file, system_config={}, collect_metadata=False)"
+    "sp = SedProcessor(runs=[44762], config=config_override, user_config=config_file, collect_metadata=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Add Jitter\n",
+    "In order to avoid artifacts arising from incommensurate binning sizes with those imposed during data collection, e.g. by the detector, we jitter all the digital columns."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sp.add_jitter()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### inspect the dataframe\n",
+    "Looking at the dataframe can give quick insight about the columns loaded and the data available. \n",
+    "- `sp.dataframe` shows the structure of the dataframe without computing anything. Interesting here are the columns, and their type.\n",
+    "- The `sp.dataframe.head()` function accesses the first 5 events in the dataframe, giving us a view of what the values of each column look like, without computing the whole thing.  `sp.dataframe.tail()`does the same from the end. \n",
+    "- `sp.dataframe.compute()` will compute the whole dataframe, and can take a while. We should avoid doing this."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sp.dataframe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sp.dataframe.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Binning\n",
+    "Here we define the parameters for binning the dataframe to an n-dimensional histogram, which we can then plot, analyse or save.\n",
+    "\n",
+    "If you never saw this before, the type after `:` is a \"hint\" to what type the object to the left will have. We include them here to make sure you know what each variable should be.\n",
+    "```python\n",
+    "a:int = 1 # a is an integer\n",
+    "b:float = 1.0 # b is a float\n",
+    "c:str = 1 # we hint c to be a string, but it is still an integer\n",
+    "```\n",
+    "This is totally optional, but can help you keep track of what you are doing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the name of the axes on which we want to bin\n",
+    "axes: List[str] = ['dldPosY', 'dldPosX']\n",
+    "# the number of bins for each axis\n",
+    "bins: List[int] = [480, 480]\n",
+    "# for each axis, the range of values to consider\n",
+    "ranges: List[List[int]] = [[420,900], [420,900]]\n",
```

Comment:
Strictly speaking, they should be a list of tuples of float, but this works off course.